scraping/web_scraping.R

```diff
@@ -57,4 +57,33 @@ for(i in 1:nrow(df)){
   
   
   df$Education[i]<-education_data 
-}
```

Comment:
delete } and newline

Reply:
add an ifelse function for Universities starting with the word "University" 
education_data<- ifelse(grepl("^University", biography_data) == TRUE, gsub("Personal.*", " ", biography_data), gsub("University.*", "University", biography_data))